operations/helm/charts/mimir-distributed/templates/querier/querier-so.yaml

```diff
@@ -44,4 +44,15 @@ spec:
       {{- end }}
     name: cortex_querier_hpa_default_requests_duration
     type: prometheus
-{{- end }}
\ No newline at end of file
+  {{- if .Values.querier.kedaAutoscaling.predictiveScalingEnabled }}
+  - metadata:
+      query: sum(max_over_time(cortex_query_scheduler_inflight_requests{container="query-scheduler",namespace="{{ .Release.Namespace }}",quantile="0.5"}[30m] offset 6d23h30m))
```

Comment:
Shouldn't this use the values of `predictiveScalingPeriod` and `predictiveScalingLookback`?

Reply:
it is now